sdcm/results_analyze/__init__.py

```diff
@@ -96,9 +96,21 @@ def _test_version(self, test_doc):
                 key = test_doc['_source']['versions'].get(value)
                 if key:
                     return key
+        else:
+            self.log.error('Scylla version is not found for test %s', test_doc['_id'])
+            return ''
+
+        res = dict()
+        if 'version' in test_doc['_source']['versions']:
```

Comment:
`test_doc['_source']['versions']` should include 'scylla-server' or 'scylla-enterprise-server'. If it doesn't we reach this code
with different structure, why so?

Reply:
AFAIU we don't have `scylla-enteprise-server`, as it `-enterprise` is removed by command `versions[package.replace('-enterprise', '')]` in [here](https://github.com/scylladb/scylla-cluster-tests/blob/dc1f4993e735ddc7330ba90a00596ddba9a02eef/sdcm/db_stats.py#L466)

anyway, even if we do have it, the structure inside will be the very same as without `-enterprise`

Reply:
better to have a static structure, smth. like:
```python
res = {
    "version": test_doc['_source']['versions'].get("version", ""),
    "date": "",
    "commit_id": "",
    "build_id": ""
}
```

Reply:
done